src/VisualStudio/CSharp/Impl/Options/AutomationObject.cs

```diff
@@ -159,8 +159,15 @@ public int Indent_Braces
 
         public int Indent_CaseContents
         {
-            get { return GetBooleanOption(CSharpFormattingOptions.IndentSwitchCaseSection); }
-            set { SetBooleanOption(CSharpFormattingOptions.IndentSwitchCaseSection, value); }
+            get
+            {
+                return (int)_workspace.Options.GetOption(CSharpFormattingOptions.IndentSwitchCaseSection);
```

Comment:
I agree that there is a risk if the value was somehow set to two. Not sure how to get around that without setting the new option to a ridiculous value like 1000 or -1.  I did ensure that the existing defined values of 0 and 1 map to the same values going forward.